tailcfg/tailcfg.go

```diff
@@ -2883,3 +2883,9 @@ type EarlyNoise struct {
 // For some request types, the header may have multiple values. (e.g. OldNodeKey
 // vs NodeKey)
 const LBHeader = "Ts-Lb"
+
+// ServiceIPMappings maps service names (strings that conform to
+// [CheckServiceName]) to lists of IP addresses. This is used as the value of
+// the [NodeAttrServiceHost] capability, to inform service hosts what IP
+// addresses they need to listen on for each service that they are advertising.
+type ServiceIPMappings map[string][]netip.Addr
```

Comment:
can you add a little example showing some sample values to make it easier to digest?

are the IPs always subsets of your self node's Addresses? (or AllowedIPs?)

Reply:
Added clarifying comment